dashboard/app/assets/stylesheets/application.scss

```diff
@@ -2919,6 +2919,19 @@ $modal-no-icon-gap: 22px;
 
 .instructions-markdown {
   padding-top: 19px;
+
+  .instructor-image {
+    float: left;
+    padding-right: 10px;
+  }
+}
+
+html[dir="rtl"] .instructions-markdown {
+  .instructor-image {
```

Comment:
nit: this selector can be simplified to `html[dir="rtl"] .instructions-markdown .instructor-image`